tools/ion-hash

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Let's follow Go convention here and put this package under a `cmd` folder. We did something similar for ion-go https://github.com/amzn/ion-go/tree/master/cmd/ion-go if you want to use it as a reference, 

I think we can do away with bash script too but not sure if we needed it to integrate with the testing driver.

Reply:
Updated